packages/playwright-chromium/index.d.ts

```diff
@@ -18,3 +18,5 @@ import * as types from './types/types';
 
 export * from './types/types';
 export const chromium: types.BrowserType<types.ChromiumBrowser>;
+export const firefox: types.BrowserType<types.FirefoxBrowser>;
+export const webkit: types.BrowserType<types.WebKitBrowser>;
```

Comment:
can we now just copy both `index.d.ts` and `index.mjs` from `private-internal` in all the packages? They should not be different any more.